### DISCUSSION TITLE
Problemas com a posição de divs

### DISCUSSION THREAD

**Discussion by AndreRomoaldo** [frontendbr/forum#1943] [Perguntas e Respostas] [answered]
Olá pessoal como estão? Estou iniciando na profissão, e ainda tenho muitas duvidas referentes aos codigos ; - ;
Vou colocar um link do meu github, que eu postei uma issue de um site que estou montando como treinamento, alguém pode me dar um suporte?
https://github.com/AndreRomoaldo/SPAI-ITBA

Desde já obrigadão pessoal!

**Comment by wilsonneto-swe** [comment, marked as answer]
Bom dia!

Dá uma olhada no PR: [https://github.com/AndreRomoaldo/SPAI-ITBA/pull/2](https://github.com/AndreRomoaldo/SPAI-ITBA/pull/2)
Preview: [https://wilsonneto-dev.github.io/SPAI-ITBA/index.html](https://wilsonneto-dev.github.io/SPAI-ITBA/index.html)

Obs.:
- Percebi que havia um "float" no CSS, atualmente há o flexbox, não é mais necessário usar o float, é um consenso tentar evitar usá-lo.
- Pense em seu layout como caixas, onde haverá divisão entre colunas ou linhas, aplique estas divisões com flexbox, a maneira mais simples que penso em layout.

Forte abraço!
Qualquer dúvida, manda ai.

**Comment by theandersonn** [comment]
@AndreRomoaldo Blz!?

Uma dica para melhorar o fluxo. 

Poste o conteúdo de sua dificuldade junto com a issue que você abrir com um pequeno resumo, pode ser um print da interface e um trecho do código. Uma narrativa do que você tentou fazer e onde travou ajudará também.

Facilitará, porque a pessoa não precisará rodar o seu projeto e a resolução ficará arquivada para o futuro.

**Reply by AndreRomoaldo** [reply]
Okay anderson! Obrigado pela direção, sou novato então algumas coisas ainda estão fora de ordem aheuaheua
